include/osmium/io/detail/pbf_input_format.hpp

```diff
@@ -183,7 +183,7 @@ namespace osmium {
                         PBFDataBlobDecoder data_blob_parser{std::move(input_buffer), read_types(), read_metadata()};
 
                         if (osmium::config::use_pool_threads_for_pbf_parsing()) {
-                            send_to_output_queue(osmium::thread::Pool::instance().submit(std::move(data_blob_parser)));
+                            send_to_output_queue(osmium::thread::Pool::default_instance().submit(std::move(data_blob_parser)));
```

Comment:
Hmm, I'm trying to look closer and I can't see anything broken, comparing to the file in the current master.

![pbf_input_format hpp](https://user-images.githubusercontent.com/80741/28168450-715db302-67df-11e7-9c1f-1e6d84908c6a.png)